python/ql/src/semmle/python/objects/ObjectAPI.qll

```diff
@@ -124,6 +134,62 @@ class Value extends TObject {
     }
 }
 
+/** Numeric values (ints and floats). 
+ *  Includes those occurring in the source as a literal
+ *  or in a builtin module as a value.
+ */
+class NumericValue extends Value {
```

Comment:
I actually also had to implement `NumericValue` for one of my moderisations (https://github.com/Semmle/ql/pull/2739), which hasn't been merged yet. If you could cherry-pick my commits that adds NumericValue into this PR instead, that would be great! :+1:

Reply:
(Rasmus' PR has subsequently been merged, so a rebase onto `master` is probably all you need.)